srsenb/sib.conf.example

```diff
@@ -9,7 +9,7 @@ sib1 =
     (
         {
             si_periodicity = 16;
-            si_mapping_info = []; // comma-separated array of SIB-indexes (from 3 to 13). 
+            si_mapping_info = [7]; // comma-separated array of SIB-indexes (from 3 to 13). 
```

Comment:
could you remove scheduling SIB7 by default, since this is only needed if you run 2G alongside we don't require it by default

Reply:
Fixed - although somehow the commit logged my local user instead of my GitHub account and now the CLA assistant is in a funny state and I'm not sure how to rectify that.